### PR TITLE
Fixes non-human species in VR

### LINKED ISSUE #21843: Vox (and most likely others) die to oxygen in the air in VR. No equipment?
### Severity

Inconvenience

### Short Description

Vox do not spawn with their appropriate equipment in the funny VR room.

### Reproduction

1. Vox SigTech appears.
2. Go to enter VR
3. Model materializes
4. **Dying. No gear. Not at all.**

### Round ID

52162

### Test Merges

#21778 #21351 #21825 #21275

### BYOND Username

TheMommaDragon

### Was this done on a locally hosted or non-Yogstation hosted server?

No

### Additional information

"**THE VR IS BULLSHIT!!**" - Yaha

(Note: Can't orbit VR models.)

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

Forces your species to human unless you're a synth.

Fixes #21843

# Why is this good for the game?

Every VR mission is designed based on you being either a human or a synthetic.

# Testing

# Spriting

# Wiki Documentation

# Changelog

:cl:  
bugfix: Non-human species can now use VR without combusting
/:cl:

### CODE REVIEW

File: code/modules/VR/vr_sleeper.dm [hunk 1]
```diff
@@ -177,6 +177,8 @@
 		vr_human.undershirt = H.undershirt
 		vr_human.underwear = H.underwear
 		vr_human.updateappearance(TRUE, TRUE, TRUE)
+		if(!is_synth(vr_human) || !ishumanbasic(vr_human))
```
Comment (Moltijoe) [inline code/modules/VR/vr_sleeper.dm]: this would transform everyone, including synths and humans
a synth isn't a human
and a human isn't a synth

**Review by Moltijoe — CHANGES_REQUESTED** [review]
just have it change the offending species rather than all of them

**Comment by MommaDragon** [comment]
Honestly just make it do the "outfit character" command based on species and not just default. Turning everyone into humans/synths may or may not be offensive.

**Comment by Bibby0110** [comment]
> just have it change the offending species rather than all of them

All of them except human and synthetic are offending.

**Comment by Moltijoe** [comment]
no need for this pr with the other one

**Pull request closed without merging**